test/Jsr292/src/com/ibm/j9/jsr292/MethodHandleTest.java

```diff
@@ -1024,33 +902,25 @@ public void test_asFixedArity_SamePackage_Static() throws Throwable {
 		mh = mh.asVarargsCollector(String[].class);
 		MethodHandle mhFix = mh.asFixedArity();
 		
-		AssertJUnit.assertFalse(mhFix.isVarargsCollector());
-		AssertJUnit.assertEquals(5,mhFix.invoke(new String[]{"a","b","c","d","e"}));
+		Assert.assertFalse(mhFix.isVarargsCollector());
+		Assert.assertEquals(mhFix.invoke(new String[]{"a","b","c","d","e"}), 5);
 	}
 	
 	/**
 	 * Negative test : asFixedArity test using variable input
 	 * @throws Throwable
 	 */
-	@Test(groups = { "level.extended" })
+	@Test(expectedExceptions = WrongMethodTypeException.class, groups = { "level.extended" })
 	public void test_asFixedArity_SamePackage_Static_VarialbeInput() throws Throwable {
 		MethodHandle mh = MethodHandles.lookup().findStatic(SamePackageExample.class, "getLengthStatic", MethodType.methodType(int.class,String[].class));
 		mh = mh.asVarargsCollector(String[].class);
 		
 		MethodHandle mhFix = mh.asFixedArity();
 		
-		AssertJUnit.assertFalse(mhFix.isVarargsCollector());
-		
-		boolean wmtThrown = false;
-		
-		try {
-			AssertJUnit.assertEquals(5,mhFix.invoke("a","b","c","d","e"));
-		}
-		catch(WrongMethodTypeException e) {
-			wmtThrown = true;
-		}
+		Assert.assertFalse(mhFix.isVarargsCollector());
 		
-		AssertJUnit.assertTrue(wmtThrown);
+		Assert.assertEquals(mhFix.invoke("a","b","c","d","e"), 5);
```

Comment:
If this is expected to throw, it doesn't need the `Assert.assertEquals`.  The `mhFix.invoke` is sufficient.

Reply:
Agreed and fixed.